### PR TITLE
feat(documentation): add heading utility stories

### PR DESCRIPTION
**Pull request by oliverschuerch**

### CODE REVIEW

**Comment by swisspost-devs** [comment]
Preview environments are getting posted here, as soon as they are ready!

File: packages/documentation/src/stories/utilities/heading.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  Use the heading utilities, to match the font styling of a heading without the need of the
+  associated HTML element.
+</p>
+
+<div className="alert alert-warning">
+  <p>
+    We point out that semantic headings are important for{' '}
+    <a href="/?path=/docs/foundations-accessibility--page">accessibility</a>.
+  </p>
+  <p>Use these classes only in exceptional cases.</p>
```
Comment (gfellerph) [inline packages/documentation/src/stories/utilities/heading.stories.mdx:27]:
```suggestion
  <p>
    ```suggestion
    Never skip a heading level. Maintaining a semantically correct heading structure helps users of assistive technologies and search engines understand your content. Sometimes the correct heading level has the wrong styles for the design. In this case, you can make use of the heading level helper classes. Structure your headings semantically correct and just change the look of the text.
  </p>
  <p>
    Find more information on our{' '}
    <a href="/?path=/docs/foundations-accessibility--page">accessibility page</a>.
  </p>
```

File: packages/documentation/src/stories/utilities/heading.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+</div>
+
+## Usage on heading Elements
+
+Use the utility classes (`.h1` - `.h6`), to change the appereance of heading elements.
+
+<Canvas withSource="open">
+  <Story name="Heading Elements">
+    <h6 className="h3">Heading level 1 with h3 styles...</h6>
```
Comment (gfellerph) [inline packages/documentation/src/stories/utilities/heading.stories.mdx:36]:
```suggestion
    <h6 className="h3">Heading level 6 with h3 styles...</h6>
```

File: packages/documentation/src/stories/utilities/heading.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  <p>
+    We point out that semantic headings are important for{' '}
+    <a href="/?path=/docs/foundations-accessibility--page">accessibility</a>.
+  </p>
+  <p>Use these classes only in exceptional cases.</p>
+</div>
+
+## Usage on heading Elements
+
+Use the utility classes (`.h1` - `.h6`), to change the appereance of heading elements.
```
Comment (gfellerph) [inline packages/documentation/src/stories/utilities/heading.stories.mdx:32]:
```suggestion
Use the utility classes `h1` - `h6` to change the appereance of heading elements.
```

**Review by gfellerph — CHANGES_REQUESTED** [review]

File: packages/documentation/src/stories/utilities/heading.stories.mdx (hunk 2 above)
Comment (oliverschuerch) [reply packages/documentation/src/stories/utilities/heading.stories.mdx:39]: But why would you change this? It is a <h6> tag.

File: packages/documentation/src/stories/utilities/heading.stories.mdx [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+<p className="lead">
+  Use the heading utilities, to match the font styling of a heading without the need of the
+  associated HTML element.
+</p>
+
+<div className="alert alert-warning">
+  <p>
+    ```suggestion
+    Never skip a heading level. Maintaining a semantically correct heading structure helps users of assistive technologies and search engines understand your content. Sometimes the correct heading level has the wrong styles for the design. In this case, you can make use of the heading level helper classes. Structure your headings semantically correct and just change the look of the text.
+  </p>
```
Comment (gfellerph) [inline packages/documentation/src/stories/utilities/heading.stories.mdx:26]:
```suggestion
  <p>
    Never skip a heading level. Maintaining a semantically correct heading structure helps users of assistive technologies and search engines understand your content. Sometimes the correct heading level has the wrong styles for the design. In this case, you can make use of the heading level helper classes. Structure your headings semantically correct and just change the look of the text.
  </p>
```

**Review by gfellerph — CHANGES_REQUESTED** [review]

**Review by gfellerph — APPROVED** [review]

**Pull request merged**